### DISCUSSION TITLE
Where can I download a compiled .app to test

### DISCUSSION THREAD

**Discussion by vbnin** [IBM/mac-ibm-notifications#23] [Q&A] [answered]
Hi ! 
This tool looks great, I would like to test it on my computer but I could only manage to download the source code from Github with no .app file inside. How can I download a working application or where can I find it ?

Thanks,
Vincent

**Comment by SMartorelli** [comment, marked as answer]
Hi @vbnin we're working to provide a signed and notarised .app file to let the community use it without any other additional step but we don't have any ETA yet on the release date. In the meantime you can follow [these steps](https://github.com/IBM/mac-ibm-notifications/wiki/Archive,-sign-and-notarize-the-app) to sign and notarise the app with your Apple Developer Program certs starting from Xcode.

**Reply by vbnin** [reply]
Thanks @SMartorelli for your reply, I'll try the shared steps then :-)

**Reply by ManOjr5** [reply]
@SMartorelli By any chance do we have the notarised .app available now?

**Reply by SMartorelli** [reply]
Oh yes sure, you find it on every version release https://github.com/IBM/mac-ibm-notifications/releases
